traffic_ops/app/db/migrations/2023121910512261_add_comment_parameter.up.sql

```diff
@@ -0,0 +1,18 @@
+/*
```

Comment:
Same thing here, the timestamp `2023121910512261` is from last year and should be replaced with something more recent.